ws-client/Cargo.toml

```diff
@@ -23,11 +31,14 @@ soketto = "0.5"
 pin-project = "1"
 thiserror = "1"
 url = "2"
-webpki = { version = "0.22", features = ["std"] }
 rustls = "0.19.1"
 rustls-native-certs = "0.5.0"
 
 [dev-dependencies]
 env_logger = "0.8"
 jsonrpsee-test-utils = { path = "../test-utils" }
-tokio = { version = "1", features = ["full"] }
+
+[features]
+default = ["tokioV1"]
+tokioV1 = ["tokio1", "tokio1-rustls", "tokio1-util"]
```

Comment:
I don't really have any opinions if the `tokioV1` or `tokio1` is better for naming the features but we should stick to one naming scheme.

Either please change the `http client` feature flags to `tokioVx` or rename these to `tokioX`